students/DrewSmith/lessons/lesson02/assignment/code/charges_calc.py

```diff
@@ -0,0 +1,159 @@
+'''
+Returns total price paid for individual rentals
+
+This module reads a JSON file and expects each section to have 
+product_coderental_start, rental_end, price_per_day and units_rented elements.
+'''
+import argparse
+import json
+import datetime
+import math
+import logging
+
+
+
+def load_logging(log_level, logger=None):
+    '''
+    Loads standard logging setup
+
+    :param log_level: logging level from the user
+    (1: Error, 2: Warning, 3: Debug)
+    '''
+    log = "%(asctime)s %(filename)s:%(lineno)-3d %(levelname)s %(message)s"
+    debug_dict = {'1': logging.ERROR, '2': logging.WARNING, '3': logging.DEBUG}
+    if logger is None:
+        logger = logging.getLogger()
+    logger.setLevel(debug_dict.get(log_level, logging.CRITICAL))
```

Comment:
This is one way to handle it, but it is setting up logging. I get it is to the console and shouldn't print anything, since there aren't any critical messages, but why not just return early if we aren't logging anything?